src/SFA.DAS.Employer.PR.Domain/OuterApi/Responses/GetEmployerUserAccountsResponse.cs

```diff
@@ -1,5 +1,4 @@
 ﻿namespace SFA.DAS.Employer.PR.Domain.OuterApi.Responses;
 public record GetEmployerUserAccountsResponse(string FirstName, string LastName, string EmployerUserId, bool IsSuspended, IEnumerable<EmployerUserAccountItem> UserAccounts);
 
-public record EmployerUserAccountItem(string EncodedAccountId, string DasAccountName, string Role);
-
+public record EmployerUserAccountItem(string EncodedAccountId, string DasAccountName, string Role);
```

Comment:
I see a lot of these where the last new line has been removed. Github shows missing new line at the end, so I would like to keep these newlines.